packages/devextreme/testing/tests/DevExpress.ui.widgets.htmlEditor/htmlEditorParts/api.tests.js

```diff
@@ -519,14 +538,14 @@ testModule('API', moduleConfig, () => {
             const done = assert.async();
 
             const converter = {
-                toHtml: sinon.stub(),
-                fromHtml: sinon.stub(),
+                toHtml: sinon.stub().callsFake((e) => e),
+                fromHtml: sinon.stub().callsFake((e) => e),
             };
 
             this.options = {
                 converter,
                 onValueChanged: () => {
-                    assert.strictEqual(converter.toHtml.callCount, 0);
+                    assert.strictEqual(converter.toHtml.callCount, 1);
```

Comment:
Hm, why is it changed? We entered a character (changed html) => seems like only "fromHtml" should be called once for a value property update

Reply:
https://github.com/DevExpress/DevExtreme/pull/28318#discussion_r1840895712